Packs/ServiceNow/Integrations/ServiceNowv2/ServiceNowv2.py

```diff
@@ -1502,6 +1504,23 @@ def delete_attachment_command(client: Client, args: dict) -> tuple[str, dict, di
     raise DemistoException("Error: No record found. Record doesn't exist or ACL restricts the record retrieval.")
 
 
+def get_attachment_command(client: Client, args: dict) -> list:
+    """Retreives attachment from a ticket.
+
+    Args:
+        client: Client object with request.
+        args: Usually demisto.args()
+
+    Returns:
+        Demisto Outputs.
+    """
+    sys_id = str(args.get('sys_id', ''))
+
+    result = client.get_ticket_attachment_entries(sys_id)  # type: ignore
```

Comment:
please add an if statement to when result is empty (no attachments to retrieve):
you can use the following:
if result:
return [CommandResults(readable_output=f'Successfully retrieved attachments for ticket with sys id {sys_id}.'),result]
else:
return CommandResults(readable_output=f"Ticket with sys id {sys_id} has no attachments to retrieve.")

Reply:
I updated this myself, please let me know if this change is ok with you.
the main change is that when attachments are retrieved we return a success message and the attachments and when there are no attachment we return a message that indicate that the request was successful but there are no attachments to retrieve.